packages/client/modules/outcomeCard/components/OutcomeCard/OutcomeCard.tsx

```diff
@@ -74,7 +75,8 @@ const OutcomeCard = memo((props: Props) => {
     task,
     setEditorState,
     useTaskChild,
-    dataCy
+    dataCy,
+    disabled
   } = props
   const isPrivate = isTaskPrivate(task.tags)
   const isArchived = isTaskArchived(task.tags)
```

Comment:
we've already got an `isArchived` flag here. let's use that instead to reduce the number of variables.
consider renaming `disabled` to `isArchived`. the task itself really isn't disabled, so it might get confusing when revisiting the code later on

Reply:
Yeah that makes sense. For some reason I didn't see isArchived. I'll just use that.